tests/Kernel/fixtures/rendering.yml

```diff
@@ -1241,26 +1241,24 @@
           icon: "location"
   assertions:
     count:
-      '.ecl-page-header': 1
-      '.ecl-breadcrumb__segment': 2
+      '.ecl-page-header-core': 1
+      '.ecl-breadcrumb-core__segment': 2
       '.ecl-lang-select-page button.ecl-button': 1
       '.ecl-lang-select-page .ecl-expandable__content ul li a.ecl-link': 2
-      '.ecl-page-header__meta-list': 1
-      '.ecl-page-header__info-list': 1
-      '.ecl-page-header__info-item': 2
+      '.ecl-page-header-core__meta': 1
```

Comment:
Can we add more items? We should test the rendering for all the possible classes for breadcrumb items:
`<li class="ecl-breadcrumb-core__segment" data-ecl-breadcrumb-core-item="static">`
`<li class="ecl-breadcrumb-core__segment ecl-breadcrumb-core__segment--ellipsis" data-ecl-breadcrumb-core-ellipsis>`
`<li class="ecl-breadcrumb-core__segment" data-ecl-breadcrumb-core-item="expandable">`
`<li class="ecl-breadcrumb-core__segment ecl-breadcrumb-core__current-page" data-ecl-breadcrumb-core-item="static" aria-current="page">`